lib/grape/validations/validators/base.rb

```diff
@@ -93,3 +93,10 @@ def fail_fast?
     end
   end
 end
+
+Grape::Validations::Base = Class.new(Grape::Validations::Validators::Base) do
+  def initialize(*)
+    super
+    warn '[DEPRECATION] `Grape::Validations::Base` is deprecated. Use `Grape::Validations::Validators::Base` instead.'
```

Comment:
I'll add a PR for adding a warn message to MissingGroupType and UnsupportedGroupType